woltka/file.py

```diff
@@ -277,20 +277,20 @@ def id2file_from_dir(dir_, ext=None, ids=None):
     Notes
     -----
     Only top-level directory is searched. Only files but not subdirectories are
-    considered.
+    considered. Hidden files will be ignored.
     """
     res = {}
-    for fname in listdir(dir_):
-        if isfile(join(dir_, fname)):
+    for fname in glob.glob(join(dir_, '*')):
+        if isfile(fname):
             try:
-                id_ = file2stem(fname, ext)
+                id_ = path2stem((fname), ext)
```

Comment:
Why this line has double parentheses?

Reply:
Thanks for catching this, that was leftover from shuffling stuff around. Afterwards I found path2stem so those aren't needed - fixed.